README.md

```diff
@@ -369,3 +401,11 @@ ssh (SSH client) is a program for logging into a remote machine and for executin
 `-X` - Enables X11 forwarding which lets you run graphical applications remotely<br>
 `-t` - Forces a tty allocation even if a command is specified. This can be used to execute shell commands on the remote machine<br>
 `-Y` - Enables X11 connection forwarding and treats X11 clients as trusted.<br>
+
+### Cryptography
```

Comment:
@ankurparihar Add some more crypto commands like `md5` and `openssl` ones. Those will be very useful, especially RSA related commands in `openssl`

Reply:
Added `base32`,`base64` and `md5sum` .
But I am not familiar with `openssl`. I read stuff on internet about it but so confusing.
So leaving `openssl` for now.

Reply:
That is alright. It's confusing at first but very useful. I'll merge this for now. You can learn about `openssl` and send in another PR later.